source/install/self-managed-changelog.md

```diff
@@ -5,11 +5,121 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
-- [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
+
+## Release v6.7 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-05-16**
+
+### Compatibility
+ - Updated Chrome recommended minimum version to v100+.
```

Comment:
@nab-77 are we still aiming to update the hardware guidelines in v6.7? If so I'd propose we mention it in the changelog and upgrade notes to ensure people see it

cc// @amyblais

Reply:
In the 6.7 timeframe we are (i.e. before we hit 7.0). Can someone share the expected impact of the web client min version wrt the server requirements (i.e. if you do not use v100 or higher what is the server impact?)

Thanks.